appinfo/info.xml

```diff
@@ -22,7 +22,7 @@ Guests users can only access files shared to them and can't create any files out
 	<screenshot>https://raw.githubusercontent.com/nextcloud/guests/master/screenshots/settings.png</screenshot>
 	<screenshot>https://raw.githubusercontent.com/nextcloud/guests/master/screenshots/dropdown.png</screenshot>
 	<dependencies>
-		<nextcloud min-version="21" max-version="21" />
+		<nextcloud min-version="22" max-version="22" />
```

Comment:
either 21 to 22
or bump the minor version and make sure tere is a stable21

Reply:
I considered this but I'm not sure how this app was managed before: there are no stablexx version branches matching server versions, only stable-1.x... Should we introduce the server-based version stable branches now ?

21 to 22 also sounds good

Reply:
I've set the min version to 21 for now